plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -83,6 +85,39 @@ fu_plugin_flashrom_debug_cb (enum flashrom_log_level lvl, const char *fmt, va_li
 	return 0;
 }
 
+static void
+fu_plugin_flashrom_device_set_version (FuPlugin *plugin, FuDevice *device)
+{
+	const gchar *version;
+	const gchar *version_major;
+	const gchar *version_minor;
+
+	/* as-is */
+	version = fu_plugin_get_dmi_value (plugin, FU_HWIDS_KEY_BIOS_VERSION);
+	if (version != NULL) {
+		/* some Lenovo hardware requires a specific prefix for the EC,
+		 * so strip it before we use ensure-semver */
+		if (strlen (version) > 9 && g_str_has_prefix (version, "CBET"))
```

Comment:
why did you take this off the quirk approach?  I thought it was Lenovo specific.

Reply:
I did; I can add it back if you think it's worth it, but I really don't think any other vendor would have a CBET12345 prefix on firmware out of coincidence.

Reply:
I guess that's true.  OK no concerns.